index.html

```diff
@@ -176,50 +241,80 @@ <h3 id="matters-considered-by-the-placement-rules">Matters considered by the pla
-  <li>Generally speaking, the processing method is based on JIS X 4051 [[JIS4051]]
-    (Formatting rules for Japanese documents).
-    However, in some cases, optional steps are used.</li>
-  <li>The ruby font size is set to half of the base character’s size as a default.
-    However, the method supports using different sizes than 1/2.
-    <aside class=note title="Reference size">
-      Because the size of ruby characters used in JIS X 4051 [[JIS4051]] set to 1/2
-      there are many examples that use the size of the ruby character as the reference.
-      However, since ruby is not restricted to 1/2,
-      this document uses the size the base character as the reference.
-    </aside>
-    <aside class=note title="Size of ruby characters">
-      In movable type typesetting,
-      when ruby characters of size 3.5 points were not available,
-      based characters of 7 points where sometimes paired
-      with ruby characters of 4 points.
-      Also, ruby associated with large base characters,
-      such as those in titles,
-      are sometimes smaller than 1/2.
-    </aside>
-  </li>
-  <li>While there are cases of ruby on both sides of the base string exist,
-    the method defined here only handles ruby on one side.
-    Handling both sides is left as a future exercise.</li>
+    and will use the same logic in either case.
+    <spanclass="checkme">
+    Specifically, the center of the ruby string and of the base character 
+    string are aligned in the inline direction for mono-ruby. 
+    </span>
+    </li>
+  <li><span class="checkme">Two step processing method is taken. 
+    In the first step, processing of layout only considers about the ruby 
+    string and the base character string (collectively call both of them as 
+    the ruby block in this note), to decide relative position of the ruby 
+    string and the ruby base character string. 
+    In the second step, processing of layout decides a position of the ruby 
+    base character string in a line, with consideration of preceding and 
+    following characters.
+    On the other word, the relative position of the ruby string and the 
+    ruby base character string decided in the first step is not modified 
+    regarding of any preceding and following characters. 
+    Also, this note does not take a method to align the first or last 
+    character of the ruby base character string to the line head or the line 
+    end, by modifying the relative position of the ruby string and the ruby 
+    base character string when the ruby base character string is placed at 
+    the line head or the line end. 
+    Summarizing the above, resulting positionings by the first step are not 
+    modified by the second step at all. 
+    </span>
+    </li>
+  <li>
+    <span class="checkme">
+    Although there are some cases with multiple methods of layout in [[JLREQ]] 
+    and JIS X 4051 [[JISX4051]], this note describes only one method based on 
+    policies described above. 
```

Comment:
> Although there are some cases with multiple methods of layout in [[JLREQ]] and JIS X 4051 [[JISX4051]], this note describes only one method based on policies described above.

What about changing it to something like:

> Although there are cases where multiple ways of positioning ruby are shown in [[JLREQ]] and and [[JISX4051]], this document only describes one method based on the policies described above.

Reply:
For JIS X 4051 with a reference link, tried some way, but 

- it seems better to have full name instead of defined reference term (might be also true for JLREQ?)
- for now, escaped Japanese text is shown in escaped form (with ones started from & #) by respec, so it doesn't seem to be better way...

There is some also in original text, so let me file new issue.

Reply:
filed as #25.